release-notes.txt

```diff
@@ -1,3 +1,528 @@
+* doc: Move roadmap in from the wiki (#20122)
+
+Build System / Tooling
+----------------------
+
++ dist/testbed-support: Add openmote board (#19979)
++ dist/tools/riotboot_gen_hdr/genhdr: add update command (#20239)
+* Bump version of UF2 package (#20035)
+* examples/gnrc_border_router: enable setting ZEP topology (#20006)
+* make all: document targets for RIOT base module (#20187)
+* make help: show generate-Makefile.ci (#20186)
+* tools/zep_dispatch: enhancements and fixes (#19996)
+
+Kconfig
+-------
+
+* tests/build_system: drop Kconfig tests (#20229)
+
+Examples
+--------
+
+* examples/dtls-sock: Enable non 32-bit architectures (#20196)
+* examples/dtls-sock: make use of helper functions (#20125)
+
+Testing
+-------
+
++ tests/periph/uart_locate_pins: new test/utility app (#20253)
++ tests/pkg/minmea: adding GGA, GSA and RMC sentences to test
+  application (#20005)
++ tests/unittests: add a unit test for ztimer (#20011)
+* .murdock: drop Kconfig compile check (#20211)
+* ci: ensure "xxx succeeded" properly fail (#20288)
+* ci: make native tests show up in "tests" (#20269)
+* tests/drivers/at: fix device table overflow (#20025)
+* tests/net/gnrc_sixlowpan_frag_sfr_congure_impl: disable test on
+  native (#20151)
+* tests/periph/uart: allow to set line ending (#20128)
+* tests/pkg/minmea: fixing RMC timestamp (#20018)
+* tests/sys/psa_crypto*: Fix failing tests (#20178)
+* tests/sys/psa_crypto_se_cipher: disable test on esp32-wroom-32 (#20152)
+* tests/sys/psa_crytpo_se: disable test on esp32-wroom-32 (#20150)
+* tests/unittests: blacklist LLVM on native (#19958)
+* tests: Slightly increase coverage of fmt unittests (#20272)
+
+API Changes
+-----------
+
++ print: Added size_t print format specifier (#20194)
+* drivers/mtd: use XFA for pointers to defined MTDs (#19465)
+* drivers/periph/gpio_ll: shrink gpio_conf_t (#20236)
+* hashes/sha2{24,56}: Remove static variables from sha256 (#20116)
+* nanocoap: defuse footgun in coap_get_uri_query() (#20195)
+* nanocoap: rename coap_get_code() -> coap_get_code_decimal() (#20004)
+
+Uncategorized
+-------------
+* Fix labeler action (#20137)
+* roadmap: extend peripheral drivers section (#20136)
+
+And 52 minor changes.
+
+
+Deprecations
+============
+
+None
+
+Removals (1)
+------------
+
+- mtd/*: drop .write() if .write_page() is implemented (#15380)
+
+
+Bug fixes (26)
+==============
+
+* dist/tools/uf2: add target to also copy families.json file (#20042)
+* backport_pr.py: increase entries per page for branches GET (#19985)
+* boards/esp32s3-usb-otg: fix the table of contents in documentation (#20156)
+* boards/native: only use pyterm wrapper with term target (#20264)
+* boards/nucleo-f413zh: Fix DMA mapping (#20225)
+* ci: fix murdock tests after dwq change (#20268)
+* cpu/cc2538: fix EUI provider (#20228)
+* cpu/cortexm_common: work around bug on WFI for STM32 (#20149)
+* cpu/nrf5x i2c: Set up correct interrupts after NOSTOP transmissions (#20282)
```

Comment:
```suggestion
```

All that this fix did was replace a lockup situation with a race condition. The next release will contain #20299 and #20298 which fix things properly, let's not make people retry anything that didn't work before on nRF52 I2C yet.